src/services/privateKeyUtils.ts

```diff
@@ -44,6 +94,29 @@ export async function decryptPrivateKey(privateKeyInfo: PrivateKeyInfo, mnemonic
 	throw new Error('Failed to decrypt private key')
 }
 
+/**
+ * Hack to allow singing the CSR with the RSA key we created for encryption
```

Comment:
Unless you want to showcase your singing skill in the next company meeting ;).
```suggestion
 * Hack to allow signing the CSR with the RSA key we created for encryption
```